src/app/routes/broker-ui/catalog-page/data-offer-cards/data-offer-cards.component.html

```diff
@@ -1,16 +1,36 @@
 <mat-card *ngFor="let dataOffer of dataOffers" class="asset-card">
   <ng-container *ngIf="dataOffer.asset; let asset">
-    <mat-card-header>
-      <mat-icon class="mat-icon-[40px]" mat-card-avatar>sim_card</mat-icon>
-      <mat-card-title>
-        <a
-          class="link"
-          href="javascript:void(0)"
-          (click)="dataOfferClick.emit(dataOffer)">
-          {{ asset.name }}
-        </a>
-      </mat-card-title>
-      <mat-card-subtitle>{{ asset.originatorOrganization }}</mat-card-subtitle>
+    <mat-card-header class="header-content">
+      <div class="icon-wrapper">
+        <mat-icon class="main-icon" mat-card-avatar>sim_card</mat-icon>
+        <div class="small-icon-wrapper">
+          <mat-icon
+            *ngIf="'ONLINE' === 'ONLINE'"
```

Comment:
this equates to true, probably something like this was meant `*ngIf="onlineStatus === 'ONLINE'"`

Reply:
Handled by changing Condition